docs/tutorial/tutorial_basics.rst

```diff
@@ -296,7 +314,7 @@ is equivalent to
 
    dmsp.data = vefi[(dmsp['mlat'] < 5) & (dmsp['mlat'] > -5)]
 
-See the :any:`Instrument` section for more information.
+See the :any:`pysat.Instrument` section for more information.
```

Comment:
```suggestion
See the :ref:`api-instrument` section for more information.
```

Reply:
you got it! I did also try a :ref: and included the reference I found in the registry area. It didn't work but the :any: did so I went with that one. Now that I'm looking at your change I think I forgot to remove the leading '_' 🐒